deltachat-ffi/deltachat.h

```diff
@@ -839,6 +839,42 @@ uint32_t        dc_send_msg_sync                  (dc_context_t* context, uint32
 uint32_t        dc_send_text_msg             (dc_context_t* context, uint32_t chat_id, const char* text_to_send);
 
 
+/**
+ * Send invitation to a videochat.
+ *
+ * This function reads the `basic_web_rtc_instance` config value,
+ * may check that the server is working in some way
+ * and creates a unique room for this chat, if needed doing a TOKEN roundtrip for that.
+ *
+ * After that, the function sends out a message that contains information to join the room:
+ *
+ * - To allow non-delta-clients to join the chat,
+ *   the message contains a text-area with some descriptive text
+ *   and a url that can be opened in a supported browser to join the videochat
+ *
+ * - delta-clients can get all information needed from
+ *   the message object, using eg.
+ *   dc_msg_is_videochat_invitation(), dc_msg_get_videochat_url()
+ *
+ * dc_send_videochat_invitation() is blocking and may take a while,
+ * so the UIs will typically call the function from within a thread.
+ * Moreover, UIs will typically enter the room directly without an additional click on the message,
+ * for this purpose, the function returns the message-id directly.
+ *
+ * As for other messages sent, this function
+ * sends the event #DC_EVENT_MSGS_CHANGED on succcess, the message has a delivery state, and so on.
+ * The recipient will get noticed by the call as usual by DC_EVENT_INCOMING_MSG or DC_EVENT_MSGS_CHANGED,
+ * depending on dc_msg_is_videochat_invitation(), however, UIs might some things differently,
+ * eg. play a different sound.
+ *
+ * @param context The context object.
+ * @param chat_id The chat to start a videochat for.
+ * @return The id if the message sent out
+ *     or 0 for errors.
+ */
+int dc_send_videochat_invitation (dc_context_t* context, int chat_id);
```

Comment:
`chat_id` is `uint32_t` everywhere else, let's not introduce legacy variable-size C types.